src/functions-and-arrays.js

```diff
@@ -1,19 +1,64 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers (num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else if (num2 > num1) {
+    return num2
+  } else {
+    return num1, num2
+  }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+function findLongestWord (words) {
+  let longestWord = "";
+  console.log(longestWord)
+  if (words.length === 0) return null;
+  if (words.length === 1) return words[0];
+  for (let i = 0; i < words.length; i++) {
+    if (longestWord.length < words[i].length) {
+      longestWord = words[i];
+    }
+  }
+  return longestWord 
+}
```

Comment:
looks good! Do you need the `if (words.length === 1) return words[0];`? might be able to refactor without